idd/Energy+.idd.in

```diff
@@ -4862,11 +4862,18 @@ Schedule:File,
        \key Yes
        \key No
        \default No
-  N4 ; \field Minutes per Item
+  N4 , \field Minutes per Item
        \note Must be evenly divisible into 60
        \type integer
        \minimum 1
        \maximum 60
+  A6 ; \field Adjust Schedule for Daylight Savings
+       \note Turn ON to adjust the schedule for DST
+       \note Turn OFF to use the schedule directly (default)
```

Comment:
Since the key choices are Yes/No, the notes should use those words instead of Turn On/Off.

Reply:
Good point.

Reply:
Noted, thanks. Changing the notes as follows in the new version: "
\note "Yes" means include Daylight Savings Time to the schedule
\note "No" means do not include Daylight Savings Time in the schedule, instead, use the schedule directly from the Schedule:File csv (default) "